### DISCUSSION TITLE
Developers guide for getting npm link working?

### DISCUSSION THREAD

**Discussion by bweis** [primefaces/community#357] [PrimeReact] [answered]
Having some trouble and I would like to submit a PR.  Any development guide for making npm link with with the monorepo setup you have?

**Comment by habubey** [comment, marked as answer]
We do not have a special guide, we would be very happy if you could send a PR as usual below link there general instructions.

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request

**Reply by melloware** [reply]
This is now available here:https://github.com/primefaces/primereact/blob/master/DEVELOPMENT.md
